types/wordpress__data/wordpress__data-tests.tsx

```diff
@@ -13,6 +13,7 @@ interface FooBar {
     bar: number;
 }
 
+// $ExpectType Store<FooBar, AnyAction>
 data.registerStore<FooBar>('foo', {
```

Comment:
I don't know whether this is documented anywhere, but it seems preferable to me to use TypeScript syntax for checking types where possible over the expected type comments.

```suggestion
const fooStore: data.Store<FooBar, data.Action> = data.registerStore<FooBar>('foo', {
```

Reply:
I actually prefer not doing it in the way you describe because you're not left struggling what to name your variables and you're not reusing a `let`/`var` assignment.

End result is the same either way, so just a personal preference.